.pre-commit-config.yaml

```diff
@@ -59,6 +59,17 @@ repos:
         - flake8==5.0.4
         - flake8-bugbear==22.7.1
         - pandas-dev-flaker==0.5.0
+    # Added pylint, at the moment it is excluding a lot of messages
+-   repo: https://github.com/pycqa/pylint
+    rev: v2.15.3
+    hooks:
+    -   id: pylint
+        types: [python]
```

Comment:
this is already defined, you don't need it

https://github.com/PyCQA/pylint/blob/ca6648a909a7c583cc6c210bcaa126b4f6fb1f3b/.pre-commit-hooks.yaml#L5

Reply:
Removed